docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.3"
+services:
+  groupincome:
+    build: .
+    command: grunt dev
+    # TODO: don't forget to make these ports configurable and in sync with Gruntfile.js
```

Comment:
I'd suggest referencing the relevant issue here (#71), and referencing this TODO at the relevant issue, so that it really won't be forgotten and the issues can be a single source of truth for work to be done

Reply:
Done! Force-pushed an update with comment added on the next line: